android/app/build.gradle

```diff
@@ -97,6 +97,13 @@ android {
     dataBinding = true
     buildConfig = true
   }
+
+  bundle {
```

Comment:
This change can be merged separately to unblock release if necessary, pre-approved )

Maybe add a comment like "// Users are complaining that app should be re-downloaded from Play Store after changing the language"